packages/aptos-wallet-adapter/src/WalletAdapters/BaseAdapter.ts

```diff
@@ -14,9 +14,10 @@ export type Address = MaybeHexString;
 export type AuthKey = MaybeHexString;
 
 export interface AccountKeys {
-  publicKey: PublicKey | null;
+  publicKey: PublicKey | PublicKey[] | null;
   address: Address | null;
   authKey: AuthKey | null;
+  minKeysRequired?: number;
 }
```

Comment:
can we modify this to be non-breaking? i.e. add additional optional fields instead of changing type of existing fields?
```
minKeysRequired? : number;
multisigPubkeys?: PublicKey[];
```

Reply:
Thank you for reviewing the PR!

We understand your consideration. But we implement this following [Aptos documentation](https://aptos.dev/guides/building-your-own-wallet/#dapp-api) where suggests `publicKey` could be `string | string[]`. 
This update won't effect the implementation for the ones who has integrated their dApps with single-signer account wallets actually. It only requires developers who want to integrate with Blocto to handle `publicKey` being an array. I wonder if this is OK from your perspective :)

Reply:
In that case I think we'll just include this in v1.0.0. Just one more thing: can you add this adapter to the testers and try them out locally?

Reply:
Yup sure, updated here 96ead46e1febb27ae179885f4ba13fc9b1d679c2
But since Blocto only supports testnet for now, and yet we couldn't access the aptos faucet via the frontend requests cuz it needs captcha, `Transfer Token` and `Faucet` on the tester isn't working properly. They should work well once the faucet on testnet is accessible.
You can still check how it works with connecting and signing a message.